index.js

```diff
@@ -71,4 +230,9 @@ module.exports = {
     push_sample_f: lsl.lsl_push_sample_f,
     push_sample_ft: lsl.lsl_push_sample_ft,
     destroy_outlet: lsl.lsl_destroy_outlet,
+    resolve_byprop,
+    create_inlet: lsl.lsl_create_inlet,
+    pull_chunk: lsl.lsl_pull_chunk_f,
```

Comment:
why not call it `pull_chunk_f` to stay consistent with the other names (e.g. `push_sample_f`)?

Reply:
Good idea